extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -214,6 +214,27 @@ private function getThumbnail( $title ) {
 	}
 
 	/**
+	 * @desc return physicalsize thumbnail URL
+	 * @todo Special case for starwars.wikia.com,
+	 * @todo rethink approach to images as a part of https://wikia-inc.atlassian.net/browse/DAT-3075
+	 * @param $thumbnail \MediaTransformOutput
+	 * @return string thumbnail URL
+	 */
+	private function getPhysicalSizeThumbUrl( $thumbnail ) {
+		global $wgPortableInfoboxCustomImageWidth;
+
+		if ( !empty( $wgPortableInfoboxCustomImageWidth ) ) {
+			$file = $thumbnail->file;
+			$height = min( self::MAX_DESKTOP_THUMBNAIL_HEIGHT, $file->getHeight() );
```

Comment:
the `wgPortableInfoboxCustomImageWidth` will be probably set to be bigger than our current 270px. Thus, if we want to scale it height proportionally instead of use `MAX_DESKTOP_THUMBNAIL_HEIGHT`, it would have to be bigger than this. I'm not sure if images taller than our current `MAX_DESKTOP_THUMBNAIL_HEIGHT` (500px) are something we want to have.